mycroft/configuration/mycroft.ini

```diff
@@ -93,3 +93,10 @@ max_time = 600  # in seconds
 notify_delay = 5  # in seconds
 rate = 16000
 channels = 1
+
+[PhillipsHueSkill]
+verbose = False
```

Comment:
I'm not super strongly against it. I'll do some testing with other team members to see what we think.